src/snackbar/snackbar-element.js

```diff
@@ -81,7 +81,9 @@ export default function SnackbarElement({
   React.useEffect(() => {
     if (__BROWSER__) {
       const observer = new window.ResizeObserver(([entry]) =>
-        setRootWidth(entry.contentRect.width),
+        window.requestAnimationFrame(() =>
```

Comment:
I'm seeing jittery performance in the snackbar. Do you see as well? https://deploy-preview-4069--baseweb.netlify.app/?story=snackbar--provider-overrides

Reply:
Yea, I see it in this example now too. Without RAF, it was throwing a runtime error: https://stackoverflow.com/questions/49384120/resizeobserver-loop-limit-exceeded

Probably because setRootWidth is too expensive operation?

Reply:
hm, actually seeing this error even with RAF in this specific example